ember-toucan-core/src/template-registry.ts

```diff
@@ -1,5 +1,7 @@
 import type ButtonComponent from './components/button';
+import type FieldComponent from './components/form/field';
 
 export default interface Registry {
   Button: typeof ButtonComponent;
+  'Form::Field': typeof FieldComponent;
```

Comment:
Admittedly still trying to wrap my head around the registry a bit, but I assume this template registry is for consumers of the add-on, like the `docs-app` and other apps+packages that'd consume this.

Adding this here, when I'd create a component inside our add-on using `Form::Field` I'd still get the glint error:

```bash

src/components/form/test.hbs:1:1 - error TS7053: Unknown name 'Form::Field'. If this isn't a typo, you may be missing a registry entry for this value; see the Template Registry page in the Glint documentation for more details.
  Element implicitly has an 'any' type because expression of type '"Form::Field"' can't be used to index type 'Globals'.
    Property 'Form::Field' does not exist on type 'Globals'.

  1 <Form::Field as |field|>
    ~~~~~~~~~~~~~~~~~~~~~~~~
  2   <field.Label for={{field.id}}>{{@label}}</field.Label>
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
... 
 21   {{/if}}
    ~~~~~~~~~
```

Reply:
correct, this is what folks would import to add to their registry, like you would with unpublished-development-types (you could / should consume this template-registry in there)

Reply:
Perfect, thank ya